third_party/blink/renderer/core/brave_page_graph/page_graph.cc

```diff
@@ -1069,7 +1069,7 @@ void PageGraph::AddGraphItem(std::unique_ptr<GraphItem> graph_item) {
       edges_.push_back(graph_edge);
     }
   } else {
```

Comment:
i'd just remove `else {}`

Reply:
then you should add return to each if statement